### ISSUE TITLE
Feedback from IETF109 - Morton comment on coding above transport

### ISSUE THREAD

**Issue by NicoKos** [irtf-nwcrg/draft-irtf-nwcrg-coding-and-congestion-in-transport#50]
The case of coding above transport should be explicit on the document that it may not be relevant since the recovered symbols would not be known to the transport.
Maybe mention cases where UDP is blocked by firewalls and TCP fall back is introduced?

**Comment by NicoKos** [comment]
Done in #54

**Comment by elochin** [comment]
Coding above transport is relevant : AL-FEC is above transport. May be you mean "out of scope" of the present document? Or AL-FEC schemes are not considered here because it does not impact on CC?

**Comment by NicoKos** [comment]
The text does not mention the 'relevant' : '<t>This approach requires that the transport protocol does not implement a fully reliable data transfer service (e.g., based on lost packet retransmission). QUIC with unreliable datagram extension <xref target="I-D.ietf-quic-datagram"></xref> is an example of a protocol for which this approach is relevant. In cases where QUIC traffic is blocked and a fallback to TCP mechanism is proposed, there is a risk for bad interactions between the TCP reliable mechanisms and coding schemes. For reliable transfers, coding usage does not guarantee better performance and would mainly reduce goodput for large file transfers. Moreover, the recovered symbols may not be known to the transport.</t'
